### PR TITLE
nmcli-device: add wifi show-password example

### PR DESCRIPTION
**Pull request by dreua**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/nmcli-device.md [hunk 1]
```diff
@@ -13,3 +13,7 @@
 - Connect to the Wi-Fi network with a specified name and password:
 
 `nmcli device wifi connect {{ssid}} password {{password}}`
+
+- Print password and QR code for the current Wi-Fi:
```
Comment (bl-ue) [inline pages/linux/nmcli-device.md:17]: Would this work?
```suggestion
- Print password and QR code for the current Wi-Fi network:
```

**Review by bl-ue — COMMENTED** [review]
Thank you for the page update @dreua!  I left a comment below for you to review.

File: pages/linux/nmcli-device.md (hunk 1 above)
Comment (dreua) [reply pages/linux/nmcli-device.md]: Of course, that's fine with me.

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
I think you're a new contributor @dreua, but did you not have to sign the CLA?  I don't see a comment from @CLAassistant 🤔

**Comment by dreua** [comment]
You are right and I signed it as instructed in the contribution guidelines. I signed it first and then opened this PR. Should I do it again?

**Comment by bl-ue** [comment]
Nope!  I didn't know that you did that.  That's perfectly fine 👍🏻

**Comment by dreua** [comment]
Alright thanks for the review :)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**